src/_includes/docs/install/accordions/install-chrome-from-cli.md

```diff
@@ -0,0 +1,14 @@
+
+<details markdown="1">
```

Comment:
What's an "accordian"? Why is this directory called that?

Reply:
This is something that is hidden under

<details>
<summary>an accordion</summary>
Like this
</details>